app/spotify-tracker/view.js

```diff
@@ -0,0 +1,25 @@
+export default class View {
+  constructor(model) {
+    this.model = model;
+    this.track = document.createElement('div');
+    this.track.classList.add('panel-block');
+    this.track.classList.add('track');
+    this.track.innerHTML = `
+      <img src="" class="track__img" alt="">
+      <div class"panel-block__content">
+      <h2 class="track__artist"></h2>
+      <h4 class="track__name"></h4>
+      </div>
+      <div class="spotify">
```

Comment:
This should be something like `panel-block__right` and inside of the `track__link` you will want to have a `span.fa.fa-spotify.fa-2x` to get the spotify link showing up.